Automattic/studio
# Simplify multipart requests by dropping manually encoding

Related to https://github.com/WordPress/playground-tools/pull/261.

## Proposed Changes

- Apply the same changes that will be introduced to `wp-now` (https://github.com/WordPress/playground-tools/pull/261). This implies dropping manually encoding multipart requests, as suggested in [this discussion](https://github.com/WordPress/playground-tools/pull/203#discussion_r1598539790).

## Testing Instructions



1. Build & open the app.
2. Start a site.
3. Open the WP-admin.
4. Navigate to the Media page.
5. Upload a media file.
6. Observe the media has been uploaded correctly to the Media Library.

## Pre-merge Checklist
